integration-cli/docker_cli_build_test.go

```diff
@@ -1568,7 +1569,18 @@ func TestBuildAddBadLinks(t *testing.T) {
 	}
 	defer os.RemoveAll(tempDir)
 
-	symlinkTarget := fmt.Sprintf("/../../../../../../../../../../../..%s", tempDir)
+	var symlinkTarget string
```

Comment:
I remember also there was `filepath.Separator` or smth like that also. Not sure it worth it.